src/terrain/chunk.hpp

```diff
@@ -135,8 +147,19 @@ class Chunk : public voxel_utility::VoxelBase {
     void R_merge(NodeGroup& g1, std::set<NodeGroup*>& to_merge);
 };
 
+/**
+ * @brief ChunkData save the voxel color id data such that is can be rapidly accessed.
+ *
+ * @details ChunkData is used in the mesher. Not only is the data in the chunk
+ * important, but also the data that borders this chunk. The SIZE for ChunkData
+ * is the SIZE for Chunk plus two, one for each edge.
+ *
+ * The total data saved in ChunkData is 4 * 18 * 18 * 18 ~ 23 kb, small enough
+ * for a L1 cache. (I think I'm not an EE)
```

Comment:
```suggestion
 * The total data saved in ChunkData is 4 * 18 * 18 * 18 ~ 23 kb, small enough
 * for a L1 cache. (I think I'm not an CE).
```

Yes it is. My machine has 32 KiB per core. However, for reasons that you don't care about, it really doesn't matter as long as it fits into L2 and you iterate lineally. If you do, likely the stuff at the beginning of the vector will be evicted by the time you get to the end.

![image](https://github.com/AlemSnyder/Fun-Game/assets/76668284/1d5331fc-068c-4daf-84ba-08dee297909b)